components/script/dom/htmlinputelement.rs

```diff
@@ -860,6 +871,28 @@ impl HTMLInputElement {
             target.fire_bubbling_event(atom!("change"));
         }
     }
+
+    // https://html.spec.whatwg.org/multipage/#value-sanitization-algorithm
+    fn sanitize_value(&self) {
+        let new_value = match self.type_() {
+            atom!("text") | atom!("search") | atom!("tel") | atom!("password") => {
+                DOMString::from(self.textinput.borrow().get_content()
```

Comment:
This clones the current value for no reason.

Reply:
Which method call is a clone?